CHANGELOG.md

```diff
@@ -3,6 +3,13 @@ Cloudhopper by Twitter
 
 cloudhopper-smpp
 ----------------
+## 6.0.3
+- Updated netty to 4.1.43.Final
+- Added method to send an async response pdu, which also updates the counters respectively 
```

Comment:
this would warrant a 6.1.0 imho ;)

Reply:
You're right! 
It started up as a plain implementation detail, but ended up as a backwards-compatible API change and it didn't occur to me to bump the minor version.
next time :)